formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -120,7 +120,11 @@ let x = [@attrEverything] (true && false);
 /**
  * How attribute parsings respond to other syntactic constructs.
  */
-let add = a => [@onRet] a;
+let add = a =>
+  [@onRet]
+  {
```

Comment:
What's happening here

Reply:
The input file actually has braces. The old test output was stripping the braces and now this diff preserves them.
This is what the input has:
```
let add(a) { [@onRet] a };
let add = fun(a) => [@onRet] a;
```
It doesn't distinguish between `[@a] { x }` and `{ [@a] x }` as you can see.